Assets/MixedRealityToolkit/Definitions/Devices/MixedRealityControllerMapping.cs

```diff
@@ -162,7 +162,7 @@ internal bool UpdateInteractionSettingsFromDefault()
                 MixedRealityInteractionMapping currentMapping = interactions[i];
                 MixedRealityInteractionMapping currentDefaultMapping = newDefaultInteractions[i];
 
-                if (Equals(currentMapping, currentDefaultMapping))
+                if (!Equals(currentMapping, currentDefaultMapping))
```

Comment:
I'm not sure I fully understand this change. This `Equals` check is intended to make sure we migrate preset action mappings to the equivalent new mapping, which means we do want them to be equal according to this method.

I'm writing some tests now to validate that things are working as I'm expecting.

Reply:
Ha, never mind :) realized that this change is correct when writing the tests!

Looking back a few revisions, this check was originally 

```
if (currentMapping.Id != currentDefaultMapping.Id ||
                        currentMapping.Description != currentDefaultMapping.Description ||
                        currentMapping.AxisType != currentDefaultMapping.AxisType ||
                        currentMapping.InputType != currentDefaultMapping.InputType ||
                        currentMapping.KeyCode != currentDefaultMapping.KeyCode ||
                        currentMapping.AxisCodeX != currentDefaultMapping.AxisCodeX ||
                        currentMapping.AxisCodeY != currentDefaultMapping.AxisCodeY ||
                        currentMapping.InvertXAxis != currentDefaultMapping.InvertXAxis ||
                        currentMapping.InvertYAxis != currentDefaultMapping.InvertYAxis)
```

Reply:
Good catch!